src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/CircleSelectionRenderer.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import React from 'react';
+import type {Normalizer, SelectionData} from './types';
+import ModifiableCircle from './ModifiableCircle';
+import PositionNormalizer from './normalizers/PositionNormalizer';
+import RoundingNormalizer from './normalizers/RoundingNormalizer';
+import SizeNormalizer from './normalizers/SizeNormalizer';
+import withPercentageValues from './withPercentageValues';
+
+type Props = {
+    containerHeight: number,
+    containerWidth: number,
+    disabled: boolean,
+    label?: string,
+    maxRadius: number | typeof undefined,
+    minRadius: number | typeof undefined,
+    onChange: (value: ?SelectionData) => void,
+    resizable: boolean,
+    round: boolean,
+    skin: 'filled' | 'outlined',
+    usePercentageValues: boolean,
```

Comment:
This prop shouldn't be necessary here anymore as well, right?

Reply:
Not really, but it will be passed through by `withPercentageValues`, so it's available here

Reply:
Additionally it enables auto completion for that prop when creating a `<CircleSelectionRenderer />` tag